runtime/src/xcm_teleport.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2025, Horizen Labs, Inc.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
```

Comment:
maybe you need the no_std directive here

Reply:
Why, it's a module not a crate. `#![no_std]` should go just in the crate `lib.rs` entrypoint.

Or I missed something?